modules/virt-node-placement-rule-examples.adoc

```diff
@@ -0,0 +1,180 @@
+  workloads:
+    nodePlacement:
+      nodeSelector:
+        example.io/example-workloads-key: example-workloads-value <2>
+----
+<1> Infrastructure resources are placed on nodes labeled `example.io/example-infra-key = example-infra-value`.
+<2> workloads are placed on nodes labeled `example.io/example-workloads-key = example-workloads-value`.
+
+.Example `HyperConverged` object with `affinity` rule
+[source,yaml,subs="attributes+"]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: {CNVNamespace}
+spec:
+  infra:
+    nodePlacement:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: example.io/example-infra-key
+                operator: In
+                values:
+                - example-infra-value <1>
+  workloads:
+    nodePlacement:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: example.io/example-workloads-key <2>
+                operator: In
+                values:
+                - example-workloads-value
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            preference:
+              matchExpressions:
+              - key: example.io/num-cpus
+                operator: Gt
+                values:
+                - 8 <3>
+----
+<1> Infrastructure resources are placed on nodes labeled `example.io/example-infra-key = example-value`.
+<2> workloads are placed on nodes labeled `example.io/example-workloads-key = example-workloads-value`.
+<3> Nodes that have more than eight CPUs are preferred for workloads, but if they are not available, pods are still scheduled.
+
+.Example `HyperConverged` object with `tolerations` rule
+[source,yaml,subs="attributes+"]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: {CNVNamespace}
+spec:
+  workloads:
+    nodePlacement:
+      tolerations: <1>
+      - key: "key"
+        operator: "Equal"
+        value: "virtualization"
+        effect: "NoSchedule"
+----
+<1> Nodes reserved for {VirtProductName} components are labeled with the `key = virtualization:NoSchedule` taint. Only pods with matching tolerations are scheduled on these nodes.
+
+[id="hostpathprovisioner-object-node-placement-rules_{context}"]
+== HostPathProvisioner object node placement rule example
+
+You can edit the `HostPathProvisioner` object directly or by using the web console.
+
+[WARNING]
+====
+You must schedule the hostpath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner cannot run. You will not be able to run virtual machines.
```

Comment:
```suggestion
You must schedule the `HostPathProvisioner` and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the `HostPathProvisioner` cannot run. It is not possible to run virtual machines.
```

Reply:
I think `HostPathProvisioner` needs to be in backticks. And also, switching the language here so it isn't future possible.